### PR TITLE
[MOBILE] Fix bugs in CI/CD for android production 

### PR DESCRIPTION
**Pull request by Mnoble-19**
#### Summary of Changes (What does this PR do?)

- Resolves CI/CD issues for android production track

#### Status of maturity (all need to be checked before merging):

- [x] I've tested this locally
- [x] I consider this code done
- [x] This change ready to hit production in its current state
- [x] The title of the PR states what changed and the related issues number (used for the release note).
- [ ] I've included issue number in the "Closes #ISSUE-NUMBER" part of the "[What are the relevant tickets?](#what-are-the-relevant-tickets)" section to [link the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
- [ ] I've updated corresponding documentation for the changes in this PR.
- [ ] I have written unit and/or e2e tests for my change(s).

#### How should this be manually tested?
- To test locally, update the app version, cd into the android directory, and run `bundle exec fastlane production` ensure to have the playstore service account in your environment variables
- The github action can only be tested after the PR has been merged

#### What are the relevant tickets?

- Closes #<enter issue number here>
- [Jira_card_number]() (If exists)

#### Screenshots (optional)
Screenshot 2023-04-01 at 13 58 59

### CODE REVIEW

**Review by Baalmart — APPROVED** [review]
Looks good to me, thanks @Mnoble-19

**Review by BenjaminSsempala — APPROVED** [review]
LGTM

File: mobile/pubspec.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 # Read more about iOS versioning at
 # https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
 # android test
-version: 2.0.17+20033
+version: 2.0.19+20036
```
Comment (NoahNsimbe) [inline mobile/pubspec.yaml:17]: Hi @Mnoble-19 Isn't the next version supposed to be 2.0.18 and code 20034??

File: mobile/pubspec.yaml [hunk 2]
```diff
@@ -14,7 +14,7 @@ publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 # Read more about iOS versioning at
 # https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
 # android test
-version: 2.0.17+20033
+version: 2.0.20+20037
```
Comment (NoahNsimbe) [inline mobile/pubspec.yaml:17]: Hi @Mnoble-19 why have we skipped `2.0.18`, `2.0.19`, `20034`, `20035` and `20036`?

**Pull request merged**